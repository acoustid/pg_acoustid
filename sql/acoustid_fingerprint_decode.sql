SELECT acoustid_fingerprint_decode('');
SELECT acoustid_fingerprint_decode('AQAAAA');
SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMm');
SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMmSCD_uA4AQ');
SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMmSCD_uA4AQAQ');
