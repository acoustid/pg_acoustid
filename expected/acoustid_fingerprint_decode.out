SELECT acoustid_fingerprint_decode('');
ERROR:  Invalid fingerprint (shorter than 4 bytes)
SELECT acoustid_fingerprint_decode('AQAAAA');
 acoustid_fingerprint_decode 
-----------------------------
 {}v1
(1 row)

SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMm');
ERROR:  Invalid fingerprint (too short, not enough input for normal bits)
SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMmSCD_uA4AQ');
ERROR:  Invalid fingerprint (too short, not enough input for exceptional bits)
SELECT acoustid_fingerprint_decode('AQAAA0mSJJGSJMmSCD_uA4AQAQ');
    acoustid_fingerprint_decode    
-----------------------------------
 {198178303,198047167,164426175}v1
(1 row)

